submissions/krylenger/html-css-popup/style.css

```diff
@@ -0,0 +1,253 @@
+* {
+    font-family: monospace, Helvetica, sans-serif, Arial;
+    font-size: 17px;
+    font-weight: 500;
+    margin: 0px;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+}
+
+a {
+    text-decoration: none;
+    outline: none;
+}
+
+img {
+    width: 75%;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+span {
+    text-align: center;
+}
+
+label {
+    width: 100%;
+}
+
+label:hover {
+    cursor: pointer;
+}
+
+.navigation-sub-menu .checkbox {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  background-color: #eee;
+  opacity: 0;
+}
+
+.checkbox:hover {
+    cursor: pointer;
+}
+
+.navigation-list-icons {
+    background-size: 35px 35px;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 100%;
+}
+
+.toggle-pop-up-label {
+    background-image: url(./icons/menu.png);
+}
+
+.navigation-notification {
+    background-image: url(./icons/notification.png);
+}
+
+.navigation-profile {
+    background-image: url(./icons/profile.jpg);
+}
+
+.toggle-pop-up:focus + label {
+    outline: 2px solid rgb(77, 77, 77);
+}
+
+.navigation-list > li > a:focus {
+    outline: 2px solid rgb(77, 77, 77);
+}
+
+.navigation-container { 
+    display: flex;
+    width: 100%;
+    height: 50px;
+    flex-direction: row;
+    justify-content: flex-end;
+}
+
+.navigation-list-container {
+    width: 320px;
+    height: 100%;
+}
+
+.pop-up-container {
+    display: none;
+    text-align: center;
+    height: 420px;
+    width: 270px;
+    border: solid 1px lightgrey;
+    border-radius: 5px;
+    box-shadow: 0px 0px 5px 0px rgba(0,0,0,1);
+}
+
+.pop-up-icons-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-content: center;
+    height: 94%;
+}
+
+ul {
+    display: flex;
+    justify-content: space-around;
+    margin: 0;
+    padding-inline-start: 0;
+}
+
+ul li {
+    list-style-type: none;
+    text-align: left;
+}
+
+.navigation-list > li {
+    width: 20%; 
+    display: flex;
+    justify-content: center;
+}
+
+.toggle-pop-up:checked ~ .toggle-pop-up-label::before {
```

Comment:
As for me, this pointer(triangle) must be a part of popup, not a part of button/label who calls it.
![image](https://user-images.githubusercontent.com/16196199/96765780-d6ab0180-13e2-11eb-8f2d-f5c6a0b833d9.png)

Reply:
when I try to attach it to pop-up-container it's being hidding from me because pop-up-container has scroll:auto to display additional links below.
What's the best practice in this case? 
Add extra container, surrounding pop-up with visible flow and attach this element to it?

Reply:
✅ done (with `position: fixed`)